src/pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -59,12 +58,11 @@ def backward(  # type: ignore[override]
             tensor: the loss value obtained from the closure
             model: the model to be optimized
             optimizer: current optimizer being used. ``None`` if using manual optimization
-            optimizer_idx: the index of the current optimizer. ``None`` if using manual optimization
             \*args: Positional arguments intended for the actual function that performs the backward, like
                 :meth:`~torch.Tensor.backward`.
             \**kwargs: Keyword arguments for the same purpose as ``*args``.
         """
-        model.backward(tensor, optimizer, optimizer_idx, *args, **kwargs)
+        model.backward(tensor, *args, **kwargs)
```

Comment:
where did optimizer go?

Reply:
This is the `LightningModule.backward()` call. Now that we don't loop over optimizers anymore, what is the point of feeding the optimizer to this hook? Is it useful? I would drop it.